keps/sig-network/1669-proxy-terminating-endpoints/README.md

```diff
@@ -252,7 +253,7 @@ gauge if a rollback is necessary.
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-Upgrade->downgrade->upgrade path has not been tested yet. We may want to require this for beta or GA.
+Upgrade->downgrade->upgrade testing (manual or automated) will be required for Beta. If tested manually, the steps will be documented in this KEP.
```

Comment:
I'm fine with this for now - but please update the KEP with it before graduating the feature to Beta in k/k (and send the PR doing that to me too).

Reply:
Yup, will do.